wiki/en/Include-QOS-Windows.md

```diff
@@ -3,13 +3,13 @@
 
 # Quality of Service (QoS)
 
-Jamulus uses DSCP/CS4 opportunistically to deal with buffer bloat, and uses a default DSCP/CS4 value of 128 (or 0x80). This is compatible with IPv4 and IPv6. Other values can be set using the -Q option, eg -Q [0..255] (where 0 disables QoS). If you want to explore the effect of non-default settings, see RFC4594. However, most people will have no need to do this.
+Jamulus uses DSCP/CS4 to opportunistically deal with buffer bloat. Setting the DS field byte to 128 (or 0x80) in a IPv4 or IPv6 packet header selects DSCP/CS4. Other byte values can be set using the -Q option, e.g. -Q [0..255] (where 0 cripples QoS). If you want to explore the effect of non-default settings, see RFC4594. However, most people will have no need to do this.
```

Comment:
I'd link DSCP/CS4 to a page describing it (i.e Wikipedia). Something not technical to allow users to decide what they do. Same with Buffer bloat. I think a site is already linked somewhere else.

Reply:
Links will eventually get broken. I think sincere users will easily find the information by searching the internet.
Note that "most people will have no need to do this" because Jamulus sets a good default value that the operating system should honor.

Reply:
Why do you think links will get broken? Wikipedia should be reliable for example and our link checker should show us that something broke.

Reply:
I'm not opposed to adding links, but I decided to mention this because there are many websites nowadays that have so many keywords linked to definitions etc. that it can actually be inconvenient. Feel free to add whatever you like because I won't get in the way.
Linking to anything that is not part of Jamulus documentation will require constant maintenance because it is not under your control. It will also be a mess to package or be moved elsewhere.

Reply:
I simplified the wording by not mentioning buffer bloat and added a link for further information. See new changes to wiki/en/Include-QOS-Windows.md above. You can resolve the conversation if this is acceptable.